### ISSUE TITLE
Scrobbling doesn't work when myplex users filter is enabled

### ISSUE THREAD

**Issue by daften** [trakt/Plex-Trakt-Scrobbler#202]
The settings were the same as before, but suddenly what I'm watching isn't shown in trakt anymore and scrobbles didn't get through. The filter just contained my plex username. When I make that field empty, everything works again.

**Comment by fuzeman** [comment]
Which version of the plugin are you using?

Could you upload your log file *(after starting to watch something in plex)*, see https://github.com/trakt/Plex-Trakt-Scrobbler/wiki/Log_Files for the location of the log files.

**Comment by daften** [comment]
Hi,

Jup, I've put it on https://gist.github.com/daften/de170a63884b173e0a13 since I can't upload files here.

Thanks!

**Comment by fuzeman** [comment]
It appears the user reported by your client is empty *(there should be a username listed in the ```''``` quotes)*:
```
2015-07-17 00:38:59,484 (1113cc000) :  INFO (logging_handler:46) - [plugin.pts.scrobbler] Ignoring item [409](...) played by filtered "scrobble_names": ''
```

Not sure what would cause this though, could you check ```Activity -> Now Playing``` lists a user when you start watching something?

For example, user should be listed like:
![](http://i.imgur.com/k15KINh.png)

**Comment by alltimemarr** [comment]
Hello,

I have the same problem. For me, it just does not scrobble at all.
I have tried all kinds of filters, my username (alltimemarr) or the shown name (Marr), and the combination.
Also I tried filtering on client. Both Plex Home Theater, and the name of my computer (like MacBook-Pro etc etc)


For me it says the same thing:
2015-08-03 22:10:51,179 (10f2df000) :  INFO (logging_handler:46) - [plugin.pts.scrobbler] Ignoring item [1149](Insurgent) played by filtered "scrobble_names": ''
2015-08-03 22:11:01,490 (10f2df000) :  INFO (logging_handler:46) - [plugin.pts.scrobbler] Ignoring item [1149](Insurgent) played by filtered "scrobble_names": ''
2015-08-03 22:11:11,779 (10f2df000) :  INFO (logging_handler:46) - [plugin.pts.scrobbler] Ignoring item [1149](Insurgent) played by filtered "scrobble_names": ''
2015-08-03 22:11:22,069 (10f2df000) :  INFO (logging_handler:46) - [plugin.pts.scrobbler] Ignoring item [1149](Insurgent) played by filtered "scrobble_names": ''
etc etc

While I did filter on my name.. Also in your screenshot, I don't see my name there.
I see the following:
http://prntscr.com/80gjen
